cooperator/readme/USAGE.rst

```diff
@@ -0,0 +1,8 @@
+A localization module is needed with this module.
```

Comment:
What happens when none is installed ? Do we get a nice error ?

Reply:
No, there is not error. 

Actually, this is part to the configuration of cooperator that is explained in the functional documentation. Since the documentation is linked in the readme, maybe this Usage file is not required. Or we could just make a usage file that points to the documentation ?

Reply:
As discussed :

- a link to the documentation is 👍 
- mentioning that localization is needed is nice
- better solution is to make localization modules `auto_install`ed